src/components/Records/Record/Organisations.vue

```diff
@@ -2,126 +2,169 @@
-              ({{ organisationLink.organisation.types.join(', ') }})
-            </span>
-          </p>
-        </div>
-
-        <div
-          v-if="organisationLink.grant"
-          class="d-flex mt-2 "
+        <v-card
+          v-if="getRelations(key).length"
+          class="pa-4 mt-15 d-flex flex-column min-height-100"
+          outlined
+          color="white"
+          tile
+          elevation="3"
         >
-          <v-tooltip top>
-            <template v-slot:activator="{ on }">
-              <v-sheet
-                class="mb-2 flag-mr"
-                v-on="on"
+          <div class="icon-container d-flex justify-center">
+            <v-icon large>
+              {{ key === 'undefined' ? $vuetify.icons.values['other_involvement'] : $vuetify.icons.values[key] }}
+            </v-icon>
+          </div>
+          <v-card-title class="pa-0 text--primary card-title-customize">
+            {{ value }}
+          </v-card-title>
+          <v-card-text
+            v-for="(organisationLink, nodeIndex) in getRelations(key)"
+            :key="'organisationLink_' + nodeIndex"
+            :class="['pl-4 pl-2 pt-0 pb-0',{'pt-8':nodeIndex===0}]"
+          >
+            <v-card
+              class="pa-4 mt-2 d-flex flex-column v-card-hover"
+              flat
+              outlined
+            >
+              <a
+                :href="organisationLink.organisation.homepage"
+                target="_blank"
+              >
+                {{ organisationLink.organisation.name }}
+              </a>
+              <p
+                v-if="organisationLink.organisation.types.length > 0 && !organisationLink.organisation.types.includes('Undefined')"
+                class="ma-0"
               >
-                <v-icon
-                  color="secondary"
-                  class="mr-2"
+                ({{ organisationLink.organisation.types.join(', ') }})
+              </p>
+              <div
+                v-if="organisationLink.grants.length"
+                class="mt-4"
+              >
+                <span v-if="organisationLink.grant">
+                  <span v-if="organisationLink.grants.length>1">grant(s):</span>
+                  <span v-else>grant:</span>
+                </span>
+                <span
+                  v-for="(grant,grantIndex) in organisationLink.grants"
+                  :key="grant+'_'+grantIndex"
                 >
-                  fas fa-funnel-dollar
-                </v-icon>
-              </v-sheet>
-            </template>
-
-            <span>Grant funding this resource</span>
-          </v-tooltip>
-          <p class="ma-0">
-            {{ organisationLink.grant.name }}
-          </p>
-        </div>
-      </v-card>
+                  <strong v-if="grant">
+                    {{ grant.name }}
+                  </strong>
+                  <span v-if="grantIndex!==organisationLink.grants.length-1"> , </span>
```

Comment:
It's a very minor thing, but would you mind removing the space before the comma, please, i.e.:

```
<span v-if="grantIndex!==organisationLink.grants.length-1">, </span>
```

Reply:
okay it will be done in next commits